src/llvm-ptls.cpp

```diff
@@ -253,8 +253,7 @@ void LowerPTLS::fix_pgcstack_use(CallInst *pgcstack, Function *pgcstack_getter,
             auto key = builder.CreateLoad(T_size, pgcstack_key_slot);
             key->setMetadata(llvm::LLVMContext::MD_tbaa, tbaa_const);
             key->setMetadata(llvm::LLVMContext::MD_invariant_load, MDNode::get(pgcstack->getContext(), None));
-            auto new_pgcstack = builder.CreateCall(FT_pgcstack_getter, getter, {key});
-            new_pgcstack->takeName(pgcstack);
+            auto new_pgcstack = builder.CreateCall(FT_pgcstack_getter, getter, {key}, "new_pgcstack");
```

Comment:
Why do we not want to take the name of the original pgcstack? Ideally if the original was unnamed, we should name the original instruction and steal the name here.